eth/catalyst/simulated_beacon_api.go

```diff
@@ -32,8 +33,9 @@ type api struct {
 
 func (a *api) loop() {
 	var (
-		newTxs = make(chan core.NewTxsEvent)
-		sub    = a.sim.eth.TxPool().SubscribeTransactions(newTxs, true)
+		newTxs   = make(chan core.NewTxsEvent)
+		sub      = a.sim.eth.TxPool().SubscribeTransactions(newTxs, true)
+		commitMu = sync.Mutex{}
```

Comment:
Just fyi, the canonical way to initialize a variable with its zero value is just to declare it, i.e. you can write

```
var (
     commitMu sync.Mutex
)
```

Initializing with a zero literal is bad style.